prosper/common/prosper_logging.py

```diff
@@ -116,19 +117,47 @@ def close_handles(self):
         for handle in self.log_handlers:
             try:
                 handle.close()
-            except Exception:   #pragma: no cover
+            except Exception:
                 warnings.warn(
                     'WARNING: unable to close logging handle',
                     RuntimeWarning
                 )
-                #TODO:testable?
                 pass #do not crash if can't close handle
 
+    def _configure_common(self, prefix, fallback_level, fallback_format, handler_name, handler):
```

Comment:
style guide: should be one-line-per-argument (and extra tab in) if the function/method takes more than 3 args.  

```python
def _configure_common(
        self, 
        prefix, 
        fallback_level, 
        fallback_format, 
        handler_name,
        handler
):
```